lua/illuminate.lua

```diff
@@ -114,7 +114,7 @@ end
 
 function M.on_attach(client)
     M.stop_buf()
-    if client and not client.supports_method('textDocument/documentHighlight') then
```

Comment:
This code path is not used. Please see the readme and update your config, you don't need to be calling internal plugin function.